### PR TITLE
Hi, I am Pritam. I have done the changes. Please check these code and file.

### PR DESCRIPTION
**Pull request by pritam-code-ai**
Hi, 
As @lucky-singh instructed me, I have done the changes and additionally, have added a new project as- tasks/speech-synthesis-and-audio-analysis. Please check these and allow me to upload the code on your repository. 

I am eagerly waiting for your reply. 

Hope to hear from you soon. 

yours sincerely, 
Pritam

### CODE REVIEW

File: tasks/natural-language-processing/sentiment-analysis/Pritam/summary_file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+The net effect is re-allocation of computational resources from less efficient parts of the network to parts of the network where they might be more useful. The shrinking phase of MorphNet is useful to produce a neural network that optimizes the cost for a specific resource. That�s precisesly why MorphNet uses an expanding phase based on a width multiplier to expand the sizes of all layers.
```
Comment (lucky-singh) [inline tasks/natural-language-processing/sentiment-analysis/Pritam/summary_file.txt]: File has not UTF-8 characters causing error running mlflow with default parameters. Could you please check. I believe quotation in "That’s" is causing problem.

File: tasks/natural-language-processing/sentiment-analysis/Pritam/deep_sentiment_analysis.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    print("-----------------------------------------")
+    
+    if not os.path.isdir("client_uri"):
+        os.makedirs("client_uri")
+        
+    mlflow.tracking.set_tracking_uri("client_uri")
+    mlflow.tracking.get_tracking_uri()
+    
+    
+    client = MlflowClient()
```
Comment (lucky-singh) [inline tasks/natural-language-processing/sentiment-analysis/Pritam/deep_sentiment_analysis.py]: Could you please update mlflow section in the code. it's giving error while running code second time onwards as experiment names are hard coded. Please refer this [MLflow example](https://github.com/mlflow/mlflow/blob/master/examples/sklearn_elasticnet_wine/train.py) and keep options required for logging as of now.
Or please do let me know if there is another way to run the task?

File: tasks/natural-language-processing/text-summarization/Pritam/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+       a integer number less than the input number of sentences in text file. 
+
+output: 
+
+   summary_file.txt 
+
+.
+
+command:
```
Comment (lucky-singh) [inline tasks/natural-language-processing/text-summarization/Pritam/README.md]: Please add `mlflow run .` example.

Optional: If possible, please try to implement [serve example](https://mlflow.org/docs/latest/quickstart.html#saving-and-serving-models)

File: tasks/natural-language-processing/text-summarization/Pritam/advanced_conditioned_summarization.py [hunk 4]
```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+
+
+
+import mlflow
+from  mlflow.tracking import MlflowClient
+import nltk
```
Comment (lucky-singh) [inline tasks/natural-language-processing/text-summarization/Pritam/advanced_conditioned_summarization.py]: Received error 
LookupError: 
**********************************************************************
  Resource 'tokenizers/punkt/PY3/english.pickle' not found.

Please add following code after nltk import to resolve the error.
```
nltk.download('punkt')
nltk.download("stopwords")
```

**Comment by pritam-code-ai** [comment]
Hi lucky singh,
I had executed the mlflow run with python script file name but it does not
work and showing errors.

I have run the code on a new repository in my github account, like mlflow
run online_repisitory_path.git but instead of running code in my activated
python environment, it created a seperate python environment with name
like- mlflow-<alphabet_and_numbers>.

As I activated this new python environment and run code, it shows mlflow
library missing. And also, I had worked on- local file system path with
various modifications but still error showing that git type is found and
local file system not detected.

So, it will be great if you do modification on one of the task topics of my
codes and run the code for local file system and/or for git path with
mlflow run and if it works, please send me the modified files with
execution commands so that I can modify my codes in other topics in the
tasks folder.

As a mention, I work on and have run my code with mlflow commands in
command prompt in anaconda python environment.

Send what you think.

I am eagerly waiting for your reply.

Hope to hear from you soon.

Yours sincerely,
Pritam

**Comment by lucky-singh** [comment]
Sure @pritam-code-ai , I'll work on one of the task for the example. I believe you don't have to create virtual environment while using mlflow, it will take care of creating virtual environment. All you need is [Conda](https://conda.io/en/latest/miniconda.html)

File: tasks/speech-synthesis-and-audio-analysis/MLproject [hunk 5]
```diff
@@ -0,0 +1,13 @@
+name: mlflow_MLproject_file
```
Comment (lucky-singh) [inline tasks/speech-synthesis-and-audio-analysis/MLproject]: @slavakurilyak , could you please suggest a category for this task? should it be fit within tasks/natural-language-processing or should we add a new category tasks/natural-language-understanding/text-to-speech? Any suggestion would help.

Comment (lucky-singh) [inline tasks/speech-synthesis-and-audio-analysis/MLproject]: @pritam-code-ai , once I send you example, please check the conde dependencies for this task as well. I am having issue in running this task.
Thanks!

**Comment by slavakurilyak** [comment]
@lucky-singh Please use [Papers With Code](https://paperswithcode.com) as a guide when naming tasks (aka machine learning models) and categories. [Text-To-Speech Synthesis](https://paperswithcode.com/task/text-to-speech-synthesis) and [Speech Synthesis](https://paperswithcode.com/task/speech-synthesis) are tasks under [Speech](https://paperswithcode.com/area/speech) (category/area).

**Comment by pritam-code-ai** [comment]
Hi lucky singh,
I am waiting for the example modified code with instructions for how to
correctly compile one of my contribution project so that I can modify my
other projects.

Hope to hear from you soon.

Yours sincerely,
Pritam

**Comment by lucky-singh** [comment]
Hi @pritam-code-ai , I have send you example as a pull request to your repository on same day of your original request.
I am sorry if you didn't get notification from github.
please have a look here.

https://github.com/pritam-code-ai/ai-platform/pull/1

**Comment by slavakurilyak** [comment]
@pritam-code-ai any updates on this pull request?

**Comment by pritam-code-ai** [comment]
Hi Slava,
I will send you the update within 3 days.

Yours sincerely,
Pritam

**Comment by slavakurilyak** [comment]
@mlaradji Please review this contribution. Note, Pritam's branch is 1 commit ahead, 128 commits behind produvia:master.

**Comment by pritam-code-ai** [comment]
Hi Slava,
have you accepted my github contribution deep learning projects python
codes ?

I am eagerly waiting for your reply.

Hope to hear from you soon.

Yours sincerely,
Pritam

**Comment by mlaradji** [comment]
Hi @pritam-code-ai. Sorry for the delay.

The branches `produvia/master` and `pritam-code-ai/master` shared no history. To make it easier to merge your changes, I created a new branch in my repository [`mlaradji/pritam-code-ai`](https://github.com/mlaradji/ai-platform/tree/pritam-code-ai) that should contain all your work. Feel free to clone that repository and modify it. The new branch will hopefully be merged in #156.

**Pull request closed without merging**